src/bin/inx-chronicle/cli.rs

```diff
@@ -231,12 +232,29 @@ impl ClArgs {
                         // Panic: Acceptable risk
                         res.unwrap()?;
                     }
-                    return Ok(true);
+                    return Ok(PostCommand::Exit);
+                }
+                #[cfg(debug_assertions)]
+                Subcommands::ClearDatabase { no_run } => {
+                    tracing::info!("Connecting to database at bind address `{}`.", config.mongodb.conn_str);
+                    let db = chronicle::db::MongoDb::connect(&config.mongodb).await?;
+                    db.clear().await?;
+                    tracing::info!("Database cleared successfully.");
+                    if *no_run {
+                        return Ok(PostCommand::Exit);
+                    }
```

Comment:
Instead of a `--no-run` flag I would do the opposite and add a `--run` flag, that says "please clear the database and also start Chronicle for me". That's because making it "opt-in" instead of "opt-out" is probably more what a user would expect from the name of that command.

Reply:
Well, keep in mind this is really just for us. And in my testing I have found it is most useful to run it when starting the application during debugging.

Reply:
I agree with Alex here, I wouldn't expect the command to automatically run Chronicle.

Reply:
> I agree with Alex here

You tryin to be funny, funny guy?

Reply:
:popcorn:

Reply:
I agree with both of you ❤️